src/Template/Element/packages_panel.ctp

```diff
@@ -75,3 +80,60 @@
         </section>
     <?php endif; ?>
 <?php endif; ?>
+
+<script>
+    $(document).ready(function() {
+        var baseUrl = '<?= $this->Url->build([
+            'plugin' => 'DebugKit',
+            'controller' => 'Composer',
+            'action' => 'checkDependencies'
+        ]); ?>';
+
+        var console = $('.console');
+
+        function showMessage(el, html) {
+            el.show().html(html);
+        }
+
+        function buildLoader() {
+            return '<div class="loading">Loading <?= $this->Html->image('DebugKit.cake.icon.png', ['class' => 'indicator'])?></div>';
+        }
+
+        function buildSuccessfulMessage(response) {
+            var html = '';
+            if (response.packages.bcBreaks === undefined && response.packages.semverCompatible === undefined) {
+                return '<pre class="success-message">All dependencies are up to date</pre>';
+            }
+            if (response.packages.bcBreaks !== undefined) {
+                html += '<h4>Update with potential BC break</h4>';
+                html += '<pre>' + response.packages.bcBreaks + '</pre>';
```

Comment:
I'm actually using the tags issued by the console output to style the result so escaping here would break that.